roles/ironic-common/templates/etc/ironic/ironic.conf

```diff
@@ -49,11 +55,39 @@ rabbit_userid = {{ rabbitmq.user }}
 rabbit_password = {{ secrets.rabbit_password }}
 
 [glance]
-glance_host={{ endpoints.main }}
+glance_host = {{ endpoints.main }}
```

Comment:
So in Nova, we hit glance directly to the controller and use http to get images, rather than trying to transfer images through the load balancer. Should we do the same here?